lib/absinthe/phase/document/validation/no_fragment_cycles.ex

```diff
@@ -35,7 +35,17 @@ defmodule Absinthe.Phase.Document.Validation.NoFragmentCycles do
   defp check(fragments) do
     graph = :digraph.new([:cyclic])
     try do
-      check(fragments, graph)
+      with {fragments, 0} <- check(fragments, graph) do
+        fragments = Map.new(fragments, &{&1.name, &1})
+
+        fragments =
+          graph
+          |> :digraph_utils.topsort
```

Comment:
We already create a digraph for fragment cycles validation. Should we keep around ets refs in the blueprint for reuse?